locales/en-US/app.ftl

```diff
@@ -189,6 +189,24 @@ CallTree--inlining-badge = (inlined)
 ## This is the sidebar component that is used in Call Tree and Flame Graph panels.
 
 CallTreeSidebar--select-a-node = Select a node to display information about it.
+CallTreeSidebar--call-node-details = Call node details
+
+CallTreeSidebar--traced-running-time =
+    .label = Traced running time
+CallTreeSidebar--traced-self-time =
+    .label = Traced self time
```

Comment:
I believe it would be good to have some comments to explain this concept ("traced") to localizers.
(and maybe for the others too)

For "traced" I wrote recently:

> the only thing we know for sure, is that every time a sample was taken, this call stack was present
> then we try to "guess" (if you will) how that translates to a duration
> julienw
> it's common that we can't keep up with the requested interval (1ms by default); for example on Windows this is more like 2ms despite 1ms is requested, but this might also come from the machine being overloaded or something like this.

Maybe eventually we should implement a tooltip with some explanations, showing up when hovering up a `ⓘ` icon.

Reply:
> I believe it would be good to have some comments to explain this concept ("traced") to localizers.

Sure, I added some comments. Let me know what you think.

> Maybe eventually we should implement a tooltip with some explanations, showing up when hovering up a `ⓘ` icon.

Yeah, definitely I think it would be great to have a tooltip with explanations.